README.md

```diff
@@ -107,13 +107,17 @@ For example, a file named `menu.svg` in `~/Sites/sixtwothree.org/images/icons` w
 
 To use an svgeez-generated SVG sprite file, first include the file's contents at the bottom of your HTML page.
 
-In a Rails 4 application:
+In a Rails 5 application:
```

Comment:
Since there are still plenty of Rails 4 applications out in the wild, it would be helpful to have both Rails 4 _and_ Rails 5 examples in the README. Could you retain the Rails 4 example and add your Rails 5 example?